client/app/scripts/superdesk-workspace/workspace.js

```diff
@@ -237,6 +237,34 @@
         };
     }
 
+    WorkspaceSidenavDirective.$inject = [];
+    function WorkspaceSidenavDirective() {
+        return {
+            templateUrl: 'scripts/superdesk-workspace/views/workspace-sidenav-items.html',
+            link: function(scope) {
+
+                /*
+                 * Function for showing and hiding monitoring list
+                 * while authoring view is opened.
+                 *
+                 * @param {string} location Gets default url of called element
+                 * @param {object} e Gets $event from the element
+                 */
+                scope.hideMonitoring = function (location, e) {
+                    var flags = scope.$parent.flags,
+                        route = scope.currentRoute.href;
```

Comment:
Would it make sense to make this more explicit so that it's clear where `route` and `flags` come from? E.g. by explicitly specifying the parent directive (if any) or creating a new scope, passing these two values in explicitly?